readme.md

```diff
@@ -107,6 +111,21 @@ var express = require("express");
 var app = express();
 
 // Your code starts here...
+app.get('/index', function(req,res){
+  res.send("This route will read entries from the db");
+});
+
+app.post('/new', function(req,res){
+  res.send("This route will create a new entry into the db");
+});
+
+app.put('/:id/edit', function(req,res){
```

Comment:
The "put" indicates that this is the update action and would not have the "edit" suffix in the url.